docs/accessories/developer/api_fundamentals.md

```diff
@@ -0,0 +1,40 @@
+---
+title: API Fundamentals
+project: accessories
+---
+
+### Creating an Accessory
+
+You can either create an Accessory by implementing the `Accessory` interface or by extending the `AccessoryItem` which implements the `Accessory` interface. 
+
+The first requires that you register your custom `Accessory` to the given `Item` instance using the `AccessoryRegistry#register` method while the later `AccessoryItem` will register the accessory when the class is instantiated
+
+This `Accessory` interface contains various functions that can be preformed like `tick`ing, controlling equipablity, listening to when equipment is changed, adding attribute modifiers dynamically or staticlly, adding custom tooltip info, and more.
+
+### Adding a Custom Accessory Renderer
+
+With creating a custom `Accessory` comes with the ability to add some rendering that is attached to the player. You can do such by creating a custom instance of `AccessoryRenderer` and registering such using the `AccessoriesRendererRegistry.registerRenderer` method. More information about the API available to developers can be found in the rendering API section. {TODO: WIP ADD LINK TO SUCH}
+
+!!! warning "Default Rendering Behavior"
+    By default as discussed within the Rendering API section of the docs, All Accessories will use `DefaultAccessoryRenderer` unless a custom `AccessoryRenderer` is registered or no renderer is registered using `AccessoriesRendererRegistry#registerNoRenderer` as such is meant to have some rendering.
+
+### Creating a custom Slot Predicate
+
+In cases where a Tag may not the be the best solution or using Data Components on your ItemStack may not be suitable, one can create a custom `SlotBasedPredicate` where you can have more general control of allowing `Accessory` entries from being equipped to a given slot. 
+
+This is useful for dynamically accessory types that may have multiple iterations that would make it difficult to use Tag without possible issues.
```

Comment:
the use of "dynamically" is wrong here but i'm not sure if its meant to just say "dynamic" or if there's meant to be another word between "dynamically" and "accessory"